core/src/banking_stage/vote_storage.rs

```diff
@@ -38,7 +36,6 @@ pub struct VoteStorage {
 /// multi-iterator checking function.
```

Comment:
this comment is no longer correct. search for references of multi iterator and remove comments. its not in the code any more, so we should not reference it.